docs/processes/quality-assurance/templates/bug-report-template.md

```diff
@@ -0,0 +1,27 @@
+## **Bug Report Template**
```

Comment:
**nitpick:** Since this is a header, I think we don't need emphasis (ie. `**`). There is no visual difference 🤔